src/modbus/ModbusClient.cpp

```diff
@@ -65,13 +75,14 @@ bool ModbusClient::connect()
 bool ModbusClient::disconnect()
 {
     std::lock_guard<decltype(m_modbusMutex)> l{m_modbusMutex};
-    return destroyContext();
```

Comment:
This will cause a memory leak if the context is not destroyed and new context is created on every call to connect